virttest/utils_misc.py

```diff
@@ -3025,7 +3025,7 @@ def get_ksmtuned_pid(self):
         try:
             utils_path.find_command("ksmtuned")
         except utils_path.CmdNotFoundError:
-            raise KSMTunedNotSupportedError
+            raise exceptions.TestCancel("Error, ksmtuned command not found.")
```

Comment:
Would you pls remove 'Error' in the message? It's kind of confusing to have both 'CANCEL' and 'Error' in the result. Others look good to me.

Reply:
Done!